sktime/forecasting/compose/_pipeline.py

```diff
@@ -1730,17 +1730,8 @@ def __init__(self, estimator, permutation=None, steps_arg="steps"):
         self.steps_arg = steps_arg
 
         super().__init__()
-        tags_to_clone = [
-            "ignores-exogeneous-X",  # does estimator ignore the exogeneous X?
-            "capability:insample",
-            "capability:pred_int",  # can the estimator produce prediction intervals?
-            "capability:pred_int:insample",
-            "requires-fh-in-fit",  # is forecasting horizon already required in fit?
-            "enforce_index_type",  # index type that needs to be enforced in X/y
-            "fit_is_empty",
-        ]
 
-        self.clone_tags(self.estimator, tags_to_clone)
+        self._set_delegated_tags(estimator)
```

Comment:
Small suggestion: if you move this call below `self._set_permuted_estimator()`, you can modify it to simply be `self._set_delegated_tags()` (w/out the `estimator` argument). Don't feel strongly about either way, just that it would look more consistent with the other calls